.kokoro/docs/common.cfg

```diff
@@ -58,3 +58,8 @@ before_action {
     }
   }
 }
+
+env_vars: {
+  key: "SECRET_MANAGER_KEYS"
+  value: "releasetool-publish-reporter-app,releasetool-publish-reporter-googleapis-installation,releasetool-publish-reporter-pem"
```

Comment:
Wouldn't it be simpler to just add these. values to `populate-secrets.sh`? Do we really need to use an env var?

Reply:
We can move it to `populate-secrets.sh`, but the standard way is to pass the `SECRET_MANAGER_KEYS` as an environment variable for all the client libraries. From the maintenance point of view, deviating from this might not be a good idea.